src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -53,6 +53,11 @@ pub type sctp_assoc_t = u32;
 
 pub type eventfd_t = u64;
 
+pub type nlink_t = u64;
+pub type dev_t = u64;
+pub type ino_t = c_ulong;
```

Comment:
This should be u64.  Otherwise `stat` is broken on x86.
```suggestion
pub type ino_t = u64;
```